test/helper.js

```diff
@@ -0,0 +1,3 @@
+import setupJSDom from './util/mocha-setup';
+
+beforeEach(setupJSDom);
```

Comment:
If you're doing this, should we remove the setupJSDom call that's in mocha-setup, and also remove mocha-setup from the mocha opts file?

Also, does it not work if you call beforeEach in the mocha-setup file?

Reply:
files that are required through mocha.opts don't have the mocha dsl setup yet. I founded I needed to do it this way (including keeping the setupJSDom call) because some of the api client code  references window.location on load and would fail without JSDom being setup.